third_party/2and3/simplejson/__init__.pyi

```diff
@@ -1,10 +1,17 @@
-from typing import Any, IO, Text
+import sys
+from typing import Any, IO, Text, Union
 
 from simplejson.scanner import JSONDecodeError as JSONDecodeError
 from simplejson.decoder import JSONDecoder as JSONDecoder
 from simplejson.encoder import JSONEncoder as JSONEncoder, JSONEncoderForHTML as JSONEncoderForHTML
 
+if sys.version_info >= (3, 6):
+    _LoadsString = Union[Text, bytes, bytearray]
+else:
+    _LoadsString = Text
```

Comment:
I don't think the version check is necessary. Since `simplejson` is an external library, it's not bound to the Python version used. I just tested `simplejson.loads()` with Python 3.5 and a bytes string and it worked. For Python 2, `bytes` equals `str` and is contained in `Text`, so `Union[Text, bytes]` is equivalent to just `Text` and it should also accept `bytearray`s.